src/nvim/message.c

```diff
@@ -504,6 +505,9 @@ int emsg(const char_u *s_)
     if (cause_errthrow((char_u *)s, severe, &ignore) == true) {
       if (!ignore) {
         did_emsg = true;
+        if (msg_first_ignored_err == NULL) {
+          msg_first_ignored_err = xstrdup(s);
+        }
```

Comment:
This is ugly, but there's no way around it that I could find. `call_func()` does not treat _non-exception, abort-triggering_ `emsg()` errors the same way as `do_cmdline_cmd()`, and following all of the global flags is a nightmare.

So this is a special-case global that holds the error in this very specific case.